integration/apps/build_func.sh

```diff
@@ -90,3 +95,47 @@ unpack_archive() {
         tar xvf ${ARCHIVE}
     fi
 }
+
+# Clear out previously installed binaries
+clean_geopm() {
+    local BUILD_DIR=$1
+    if [ -e "${BUILD_DIR}" ]; then
+        echo "WARNING: Previous build of geopm or other data found at ${BUILD_DIR}"
+        read -p "OK to delete all object files and recompile? (y/n) " -n 1 -r
+        echo
+        if [[ ${REPLY} =~ ^[Yy]$ ]]; then
+            rm -rf ${BUILD_DIR}
+        elif [[ ! ${REPLY} =~ ^[Nn]$ ]]; then
+            echo "Error: Did not understand reply: ${REPLY}" 1>&2
```

Comment:
might want to indicate that partial rebuild will happen in 'N' case in the warning message